lib/actions/apiV2.js

```diff
@@ -920,11 +923,33 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
           routingError,
           {
             rewritePayload: (response, dispatch, getState) => {
-              const withCollapsedShortNames =
-                response.data?.plan?.itineraries?.map((itin) => ({
+              const itineraries = response.data?.plan?.itineraries
+
+              // Convert user-selected transit modes from mode selector into modes recognized by OTP.
+              const activeModeStrings = activeModes.map(
+                (am) => SIMPLIFICATIONS[am.mode]
+              )
+
+              let filteredItineraries
```

Comment:
This can be a const, then you don't need an else block
```suggestion
              const filteredItineraries = clone(response.data?.plan?.itineraries)
```